### PR TITLE
GitHub chapter review

### PR DESCRIPTION
**Pull request by JessicaS11**
### Summary

This PR provides revisions to the collaboration/GitHub-novice chapter.

It may contribute to a few issues, but I didn't see one specifically for chapter revisions.

### List of changes proposed in this PR (pull-request)

- adds terms to glossary
- adds links to other chapters/info
- add resources for publishing software that's hosted on GitHub
- fixes typos and other grammar

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [X] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/collaboration/github-novice/github-novice-advanced.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+This resource - [GitHub for collaborative documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) - has a [step-by-step guide](https://cassgvp.github.io/github-for-collaborative-documentation/docs/tut/4-2-Make-your-Pages-site.html) of how to make GitHub pages.
 
-This resource - [Github for collaborative documentation](https://cassgvp.github.io/github-for-collaborative-documentation/) - has a step-by-step guide of how to make Github pages.
+## Link this repository with an online platform
 
-## Link this repository with an online platform such as Open Science Framework
-To make an open workflow linking all of the tools that you use, an online platform is needed.
-Open Science Framework enables researchers to add on many different tools such as Dropbox, Github and Google workspace.
+### Open Science Framework
+[Open Science Framework]() is an online platform that enables researchers to link many different tools - such as Dropbox, GitHub and Google workspace - to create a completely open workflow.
```
Comment (da5nsy) [inline book/website/collaboration/github-novice/github-novice-advanced.md:18]:
```suggestion
[Open Science Framework](https://osf.io/) is an online platform that enables researchers to link many different tools - such as Dropbox, GitHub, and Google workspace - to create a completely open workflow.
```

File: book/website/collaboration/github-novice/github-novice-features.md [hunk 2]
```diff
@@ -1,56 +1,66 @@
...
-You now have your repo set up and you probably want to know how to add more stuff to it and use some of the useful features that it contains.
+Now that your repo is set up, you are ready to add more stuff to it and use some of its useful features.
 
-## Adding more folders, files and commits
+## Adding more folders (directories), files and commits
 
-**To create a file inside a folder, you need to:**
+* You can add more files to your repo by clicking the "add file" button and either uploading files or creating a new one.
+* Each time you create a new file or upload files you will need to add a {ref}`commit message`<rr-vcs-commit-messages> to help you remember what changes you made.
+* Note that git (and thus GitHub) do not track folders/directories, so you cannot create an empty folder.
```
Comment (da5nsy) [inline book/website/collaboration/github-novice/github-novice-features.md:10]:
```suggestion
* Note that git (and thus GitHub) does not track folders/directories, so you cannot create an empty folder.
```

File: book/website/collaboration/github-novice/github-novice-features.md [hunk 3]
```diff
@@ -1,56 +1,66 @@
...
 * In the left sidebar, click “Community”.
-* Here you will find other file recommendations that you can add in your repo.
-These files are useful when working with collaborators or members in your community.
-Specifically, look at ‘contributing’ and ‘description’.
-* Whenever possible, add a few lines as ‘code of conduct’ to make your project welcoming and inclusive for others.
-* Other interesting things to look at are “Contributors” (those who contribute to your project), “Traffic” (who and when visits your project page), and “Commits” (a timeline and number of commits made on your project).
+  * Here you will find recommended files to include in your repo.
+  These files are especially useful when working with collaborators or members in your community.
+  * The "License" file is one of the most important, because it tells others how they can (or can't) use the materials in your repo.
+  * For collaboration, be sure to look at "Contributing" and "Description".
```
Comment (da5nsy) [inline book/website/collaboration/github-novice/github-novice-features.md:33]:
```suggestion
  * For collaborations, be sure to look at "Contributing" and "Description".
```

File: book/website/collaboration/github-novice/github-novice-features.md [hunk 4]
```diff
@@ -1,56 +1,66 @@
...
+  * Whenever possible, add a "Code of conduct" to make your project welcoming and inclusive for others.
+* Other interesting things to look at are “Contributors” (those who contribute to your project), “Traffic” (who and when visits your project page) and “Commits” (a timeline and number of commits made on your project).
```
Comment (da5nsy) [inline book/website/collaboration/github-novice/github-novice-features.md:35]:
```suggestion
* Other interesting things to look at are “Contributors” (those who contribute to your project), “Traffic” (who visits your project page, and when do they do so), and “Commits” (a timeline and number of commits made on your project).
```

File: book/website/reproducible-research/vcs.md [hunk 5]
```diff
@@ -13,11 +13,12 @@
...
 ## Summary
 
-Version control is an approach to record changes made in a file or set of files over time so that you and your collaborators can track their history and review any changes.
+No matter how your group is organized, the work of many contributors needs to be managed into a single set of shared working documents.
+Version control is an approach to record changes made in a file or set of files over time so that you and your collaborators can track their history, review any changes, and revert or go back to earlier versions.
 Management of changes or revisions to any types of information made in a file or project is called versioning.
 
 In this chapter, we discuss the best practices that are relevant regardless of tools.
-This practice mainly comes from managing changes in the code repositories.
+Versioning practices mainly come from managing changes in the code repositories.
```
Comment (da5nsy) [inline book/website/reproducible-research/vcs.md:21]:
```suggestion
Versioning practices mainly come from managing changes in the code repositories.
```

**Review by da5nsy — CHANGES_REQUESTED** [review]
All LGTM (with a couple of minor suggestions).
I'd like to check the netlify preview before merging, just to check that all the links work, and it looks like that might be offline right now (#2150)

File: book/website/collaboration/github-novice/github-novice-advanced.md (hunk 1 above)
Comment (JessicaS11) [reply book/website/collaboration/github-novice/github-novice-advanced.md]: Thanks for catching all these typos, @da5nsy! I agree with your use of the Oxford comma, but The Turing Way doesn't have an official policy on them (unless that's changed recently!) other than consistency, so I was going with what was already there.

Comment (da5nsy) [reply book/website/collaboration/github-novice/github-novice-advanced.md]: Fair enough!

**Review by da5nsy — APPROVED** [review]

**Pull request merged**